core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -25,161 +25,177 @@
 import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
+import org.apache.iceberg.BaseMetastoreTableOperations;
 import org.apache.iceberg.catalog.Namespace;
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.relocated.com.google.common.base.Joiner;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.base.Splitter;
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 
 final class JdbcUtil {
   // property to control strict-mode (aka check if namespace exists when creating a table)
   static final String STRICT_MODE_PROPERTY = JdbcCatalog.PROPERTY_PREFIX + "strict-mode";
 
-  // Catalog Table
-  static final String CATALOG_TABLE_NAME = "iceberg_tables";
+  // Catalog Table & View
+  static final String CATALOG_TABLE_VIEW_NAME = "iceberg_tables_views";
   static final String CATALOG_NAME = "catalog_name";
-  static final String TABLE_NAMESPACE = "table_namespace";
-  static final String TABLE_NAME = "table_name";
+  static final String TABLE_VIEW_NAME = "name";
+  static final String TABLE_VIEW_NAMESPACE = "namespace";
+  static final String TABLE_VIEW_FLAG = "type";
```

Comment:
```suggestion
  static final String TYPE_FLAG = "type";
```

or just `TYPE` maybe?

Reply:
👍